docs/architecture.md

```diff
@@ -247,44 +92,6 @@ mdbe@?+opensuse^13 => [http://downloads.mariadb.com/enterprise/WY99-BC52/mariadb
 ```
 where mdbe@? means default mariadb community version on Opensuse13 target platfrom.
 
+### Supported VM providers and boxes
```

Comment:
It seems to be wrong to have a section dedicated to a link. If no information is provided, then may be this link should be removed from the document.